app/views/groupings/show.html.erb

```diff
@@ -1,5 +1,10 @@
 <%= render 'organizations/organization_banner' %>
 
+<%= javascript_tag do %>
+    window.organization_id = "<%= @organization.slug %>";
+    window.grouping_id = "<%= @grouping.slug %>";
+<% end %>
```

Comment:
@dinever not sure how I feel about this.

Can we parse the url to get these values for us instead?